crates/viewer/re_viewer_context/src/viewer_context.rs

```diff
@@ -79,6 +80,9 @@ pub struct ViewerContext<'a> {
     /// The focused item is cleared every frame, but views may react with side-effects
     /// that last several frames.
     pub focused_item: &'a Option<crate::Item>,
+
+    /// If a selection contains any `undraggable_items`, it may not be dragged.
+    pub undraggable_items: &'a ItemCollection,
```

Comment:
So on different frames there are different items that are currently forbidden from being dragged?

Can you give an example of such an item here?
Who is responsible for checking this?

Can this be used by `handle_select_hover_drag_interactions` so we don't need to pass in the bool everywhere?

Reply:
That's the most hacky part, I'll take feedback on that.

Currently there is ONE undraggable item, and it's always the same: the root container in the blueprint viewer. The issue is that the root container is only known by "top-of-the-hierarchy" crates (`re_viewer`, etc.), and certainly not by `re_viewer_context`. I'm not really sure how to go about this in a cleaner way.

Reply:
You can at least document it here exactly what it is was introduced for, and maybe with a TODO for "this is ugly; if you have a better idea please rewrite this" :]